weave/Scripts/Grid.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using Godot;
+
+namespace weave;
+
+public class Grid
+{
+    private readonly int _nrRows;
+    private readonly int _nrCols;
+    private readonly int _width;
+    private readonly int _height;
+    private readonly float _cellWidth;
+    private readonly float _cellHeight;
+    private readonly List<List<Cell>> _cells = new();
+
+    public class Cell
```

Comment:
Can make this a struct (will get performance I believe :eyes: )

Reply:
👀